playwright.config.js

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "@playwright/test";
+
+// jest automatically sets process.env.NODE_ENV = 'test'
+// playwright does not
+// NODE_ENV is used in updateCityDetail.js to prevent the main function from running when it's being imported for testing
+process.env.NODE_ENV = "test";
+
+// Note that we may run tests by using Parcel (development) vs running on the `dist` folder.
+const port = process.env.PORT || 1234;
+const webServerCommand = port === 1234 ? "start" : "serve-dist";
+
+const baseURL = `http://127.0.0.1:${port}`;
```

Comment:
We don't need to set these yet since we don't have any tests that run the server. But good job figuring out the code! This will save time when we do add those tests.